index.php

```diff
@@ -8,15 +8,19 @@
     <div id="shuffle-container" class="grid-row hidden">
 
 <?php
-if( have_posts() ) {
+if ( have_posts() ) {
   while( have_posts() ) {
     the_post();
     $explode_content = explode( '<!--more-->', $post->post_content );
     $lightbox_image = wp_get_attachment_image_src(get_post_thumbnail_id(), 'lightbox');
+
+    $gallery = get_post_meta($post->ID, '_igv_gallery', true);
```

Comment:
Yh the thumbs no longer load in the lightbox it says in the card. Only the gallery loads if there is one (how the user is supposed to know I've no idea)